src/index.js

```diff
@@ -56,3 +60,9 @@ export const isPassword = (value) => (
     hasSpecialCharacters,
   ].every((fn) => fn(value))
 );
+
+export const isTime = (value, min = '00:00', max = '23:59') => (
+  isRegex(value, REGEX_TIME)
+  && timeToDate(value) >= timeToDate(min)
```

Comment:
avoid duplicate timeToDate of value. set a specific date to avoid possible issues calling this at 23:59:59